src/utils/Crc.cxxtest

```diff
@@ -197,3 +197,27 @@ TEST(Crc8Test, LogonExample2)
     m.update256(0x22);
     LOG(INFO, "example: 0x%02x", m.get());
 }
+
+TEST(Crc8Test, LogonExample3)
```

Comment:
There are no EXPECTs. What is the pass criteria?

Reply:
fixed above.